packages/flutter/lib/src/cupertino/text_form_field_row.dart

```diff
@@ -232,6 +233,7 @@ class CupertinoTextFormFieldRow extends FormField<String> {
                 placeholder: placeholder,
                 placeholderStyle: placeholderStyle,
                 contextMenuBuilder: contextMenuBuilder,
+                suffix: suffix,
```

Comment:
I think we should also expose `suffixMode` so the user can control the visibility.

Reply:
Sure, I'll make the necessary changes.